androidApp/src/main/java/com/example/tasky/android/agenda/components/details/DetailsRemindAtSection.kt

```diff
@@ -113,9 +97,14 @@ fun DetailsRemindAtSection(
     }
 }
 
-private fun getRemindAtTextId(time: Long): Int =
-    RemindAtType.entries.firstOrNull { it.duration.toLong(DurationUnit.MILLISECONDS) == time }?.stringId
-        ?: R.string._10_minutes_before
+private fun getRemindAtTextId(type: RemindAtType): Int =
+    when (type) {
+        RemindAtType.TEN_MINUTE -> R.string._10_minutes_before
+        RemindAtType.THIRTY_MINUTE -> R.string._30_minutes_before
+        RemindAtType.ONE_HOUR -> R.string._1_hour_before
+        RemindAtType.SIX_HOUR -> R.string._6_hours_before
+        RemindAtType.ONE_DAY -> R.string._1_day_before
+    }
```

Comment:
Here is a good place to use Philipp’s UiText class that helps you use string resources throughout your app. It allows you to pass around a regular string or a string resource without needing a context until you want to display it on the screen.

```
sealed interface UiText {
  data class DynamicString(val value: String): UiText
  class StringResource(
    @StringRes val id: Int,
    val args: Array<Any> = arrayOf()
  ): UiText

  @Composable
  fun asString(): String {
    return when(this) {
      is DynamicString -> value
      is StringResource -> stringResource(id = id, *args)
    }
  }
  
  fun asString(context: Context): String {
    return when(this) {
      is DynamicString -> value
      is StringResource -> context.getString(id, *args)
    }
  }
}
```